client/src/App.tsx

```diff
@@ -279,13 +275,13 @@ const App = () => {
     setErrors((prev) => ({ ...prev, [tabKey]: null }));
   };
 
-  const makeRequest = async <T extends z.ZodType>(
+  const makeConnectionRequest = async <T extends z.ZodType>(
```

Comment:
I question the need for this renaming. 

All through the `App.tsx` we're now calling `makeCconnectionRequest` for everything, and the naming of the function kind of sounds like we're trying to **make a request _for a_ connection** but really it means make a **request _on the_ connection**. 

We're really just trying to **make a request**, and the connection part is not relevant and confusing.

Reply:
so the only reason I did this is because there are two `makeRequest` functions in the codebase [here](https://github.com/modelcontextprotocol/inspector/blob/4053aa122d4ce5fda35014603e575702ae59c42d/client/src/App.tsx#L282) and [here](https://github.com/modelcontextprotocol/inspector/blob/4053aa122d4ce5fda35014603e575702ae59c42d/client/src/lib/hooks/useConnection.ts#L94). The latter one is being imported as `makeConnectionRequest` [here](https://github.com/modelcontextprotocol/inspector/blob/4053aa122d4ce5fda35014603e575702ae59c42d/client/src/App.tsx#L151)

Reply:
I can rename this to something else apart from `makeConnectionRequest`, something like `sendRequestToMCPServer` ?

Reply:
> I can rename this to something else apart from `makeConnectionRequest`, something like `sendRequestToMCPServer` ?
What was wrong with `makeRequest`?

Reply:
nothing wrong as such, my IDE was getting confused between both function definitions (linked in above comment) as both had same names and very similar signature.

Reply:
done